src/Storages/System/attachSystemTables.cpp

```diff
@@ -96,105 +96,105 @@ namespace DB
+    attach<StorageSystemCurrentRoles>(context, system_database, "current_roles", "Contains active roles of a current user. SET ROLE changes the contents of this table.");
+    attach<StorageSystemEnabledRoles>(context, system_database, "enabled_roles", "Contains all active roles at the moment, including current role of the current user and granted roles for current role.");
+    attach<StorageSystemSettingsProfiles>(context, system_database, "settings_profiles", "Contains properties of configured setting profiles.");
+    attach<StorageSystemSettingsProfileElements>(context, system_database, "settings_profile_elements", "Describes the content of each settings profile configured on the server. Including settings contraints, roles and users that the setting applies to and parent settings profiles.");
+    attach<StorageSystemRowPolicies>(context, system_database, "row_policies", "Contains filters for one particular table, as well as a list of roles and/or users which should use this row policy.");
+    attach<StorageSystemQuotas>(context, system_database, "quotas", "Contains information about quotas - a way how to limit resource usage over a period of time or track the use of resources.");
+    attach<StorageSystemQuotaLimits>(context, system_database, "quota_limits", "Contains information about maximums for all intervals of all quotas. Any number of rows or zero can correspond to one quota.");
+    attach<StorageSystemQuotaUsage>(context, system_database, "quota_usage", "Contains quota usage by the current user: how much is used and how much is left.");
+    attach<StorageSystemQuotasUsage>(context, system_database, "quotas_usage", "Contains quota usage by all users.");
+    attach<StorageSystemUserDirectories>(context, system_database, "user_directories", "Contains the information about configured user directories - directories on the file system from which ClickHouse server is allowed to read user provided data.");
+    attach<StorageSystemPrivileges>(context, system_database, "privileges", "Contains a list of all available privileges that could be granted to a user or role.");
+    attach<StorageSystemErrors>(context, system_database, "errors", "Contains a list of all error which have ever happened including the error code, last time and message with unsymbolized stacktrace.");
+    attach<StorageSystemWarnings>(context, system_database, "warnings", "Contains warnings about server configuration to be displayed in clickhouse-client one any is connected.");
+    attach<StorageSystemDataSkippingIndices>(context, system_database, "data_skipping_indices", "Contains all the information about all the data skipping indices in tables, similar to system.columns.");
+    attach<StorageSystemLicenses>(context, system_database, "licenses", "Сontains licenses of third-party libraries that are located in the contrib directory of ClickHouse sources.");
```

Comment:
Does is contain info about `rust` folder?

Reply:
Looks like no.
```
SELECT *
FROM system.licenses
WHERE library_name ILIKE '%rust%'

Query id: 350803e5-b53c-453f-9d7f-1c8f1ccd5d13

Ok.
```